eovpn/openvpn3_backend/dbus.py

```diff
@@ -2,6 +2,14 @@
 import logging
 from eovpn.eovpn_base import Base
 
+import sys
+import pathlib
+
+openvpn3_python_dir = pathlib.Path("/var/run/host/usr/lib/").glob("python*")
+openvpn3_python_dir = next(openvpn3_python_dir) / "site-packages"
```

Comment:
I'm not sure why you need to this kind of tricks.  And this will probably fail on Debian/Ubuntu, as they (for whatever reason those distros have) installs "external" packages under `/usr/lib/python*/dist-packages` (sigh)

Reply:
> I'm not sure why you need to this kind of tricks. And this will probably fail on Debian/Ubuntu, as they (for whatever reason those distros have) installs "external" packages under `/usr/lib/python*/dist-packages` (sigh)

Flatpak.

I had a hunch it wouldn't work on other distros (`/usr/local` is a edge cases that need to tested.). I will try something to fix it before merging this.

Reply:
fixed in 5c08f372fab07d4ba9b7d88dfb2e0c417c388494